### ISSUE TITLE
 Starting from Review Batch and then starting with the first item in the batch to mark items as reviewed, doesn't capture all items upon completion. 

### ISSUE THREAD

**Issue by Alexandermay** [curationexperts/mira#650]
1. After batch ingest, select Review Batch button.
![1](https://cloud.githubusercontent.com/assets/4700092/8329844/d8639056-1a4a-11e5-9b57-6f1ff639dc08.jpg)

2. Select the first item, and Mark it as Reviewed.  Select next.

![2](https://cloud.githubusercontent.com/assets/4700092/8329855/fdcedc7e-1a4a-11e5-959f-a5dd6bee335a.jpg)

3. When you are finished, review the batch again, and the status is still incomplete, with objects not checked as completed.  (Note, I did not see these objects when going through the above steps.)

![3](https://cloud.githubusercontent.com/assets/4700092/8329877/395804fa-1a4b-11e5-8814-a07250223f72.jpg)

Also, when you redo the steps with the old batch, it seems to now capture the items originally missed.  I can supply new items if you need it to test.

Thanks.

**Comment by Alexandermay** [comment]
Batch id 180 could be used to reduplicate the above issue.

**Comment by mark-dce** [comment]
@Alexandermay - When I go through and look at batch 176, I see items draft:858, draft:867, and draft:877 marked as not reviewed.  When I clicked review batch on this batch and clicked next though all 25 items, I saw those pids as still needing to be reviewed.  After clicking "Mark as Reviewed" on each, the batch status shows as complete.

![mira-650 - draft-858 not reviewed](https://cloud.githubusercontent.com/assets/3064318/8389748/9c45faa2-1c3c-11e5-9424-cde7b6b33231.png)

![mira-650 - draft-867 not reviewed](https://cloud.githubusercontent.com/assets/3064318/8389751/a2a3618c-1c3c-11e5-9632-29be21da1c14.png)

![mira-650 - draft-877 not reviewed](https://cloud.githubusercontent.com/assets/3064318/8389754/a865f2ce-1c3c-11e5-8dc8-fca0de22c043.png)

**Comment by mark-dce** [comment]
**UPDATE**
It appears that marking an item as "Reviewed" is updating it's relevance ranking, so that the default relevance sort is not stable when paging through a set of results in the single item views shown above.  I.E. as I mark items as reviewed in they process described above, the order of items is being altered:

In reviewing the batch below, when I reviewed item 6 - Hall of Mirrors, Palace of Versailles. and clicked next, it appeared as item 7 (and the previous item 7 was reordered as 6, so I would never see it if I just paged through "next" each time)
